bentoml/_internal/io_descriptors/multipart.py

```diff
@@ -44,28 +59,30 @@ def predict(img, annotations):
     curl -i -F img=@test.jpg -F annotations=@test.json localhost:5000/predict
     """
 
-    def __init__(self, **inputs: IODescriptor[_DescriptorType]):
+    def __init__(self, **inputs: _DescriptorType):
         for descriptor in inputs.values():
-            if isinstance(descriptor, Multipart):
+            if isinstance(descriptor, Multipart):  # pragma: no cover
```

Comment:
Why is this no cover?

Reply:
actually when writing code if users do this then they already show it to them

Reply:
no need to cover since type checker should detect this when users write their code. This acts as safety net